comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+from controls import *
+
+class Communications:
+    def __init__(self, mcuVAR: MCUInterface, controller: controls):
```

Comment:
See above comment; don't cyclically refer to classes. Don't use `controls` here.